.github/workflows/maven-ci.yml

```diff
@@ -104,16 +104,32 @@ jobs:
         uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-region: us-east-1
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Connect ECR Login to Docker
+        run: aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/s2a5w2n9
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
           platforms: linux/amd64,linux/arm64
           push: true
-          # Temporary! Update to real account
-          tags: milesgrantibigroup/otp-middleware-test:${{ github.sha }}, milesgrantibigroup/otp-middleware-test:latest
\ No newline at end of file
+          add_git_labels: true
+          tag_with_ref: true
```

Comment:
These two parameters result in [warnings in the docker build step](https://github.com/ibi-group/otp-middleware/runs/7676427873?check_suite_focus=true#step:17:1). Are they needed?